apps/ledger-live-desktop/tests/utils/xray.service.ts

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+import fs from "fs";
+
+class XrayService {
+  private static baseUrl = "https://xray.cloud.getxray.app/api/v2";
+  private static token: string;
+
+  private static async authenticate(): Promise<string> {
+    const clientId = process.env.XRAY_CLIENT_ID;
+    const clientSecret = process.env.XRAY_CLIENT_SECRET;
+
+    if (!clientId || !clientSecret) {
+      throw new Error("Missing XRAY_CLIENT_ID or XRAY_CLIENT_SECRET environment variables");
+    }
+
+    const auth: Auth = {
+      client_id: clientId,
+      client_secret: clientSecret,
+    };
+
+    try {
+      const response = await axios.post(`${this.baseUrl}/authenticate`, auth, {
+        headers: { "Content-Type": "application/json" },
+      });
+      this.token = response.headers["x-access-token"];
+      console.log(`Authentication successful`);
+      return this.token;
+    } catch (error) {
+      console.error(`Error during authentication: ${error}`);
+      throw error;
+    }
+  }
+
+  public static async importExecution(xmlFilePath: string): Promise<ImportRes> {
+    if (!this.token) {
+      await this.authenticate();
+    }
+
+    try {
+      console.log(`Importing execution from XML file: ${xmlFilePath}`);
+      const response = await axios.post(
+        `${this.baseUrl}/import/execution/junit?projectKey=B2CQA&testPlanKey=${process.env.TEST_PLAN}`,
```

Comment:
I think projectKey shouldn't be hardcoded

Reply:
I removed it, now its "hardcoded" on the workflow, so it's don't take us a input value